hieradata/site/cp/role/foreman.yaml

```diff
@@ -64,27 +40,28 @@ dhcp::pools:
     network: "139.229.164.0"
     mask: "255.255.255.0"
     gateway: "139.229.164.254"
-    range: "139.229.164.1 139.229.164.200"
+    range:
+      - "139.229.164.1 139.229.164.200"
     search_domains: "%{alias('dhcp::dnsdomain')}"
-  cds-arch:
+  cds_arch:
```

Comment:
The resource names in puppet and added to the dhcp.conf as a comment.  I would like this name to be match the subnet name in ipam and in foreman.  The foreman subnet names are largely inconsistent but I think the values in ipam are mostly in sync.  E.g. https://ipam.ls.lsst.org/index.php?page=subnets&section=1&subnetId=16

I don't understand the motivation for converting `-`'s into `_` here but if we are going to do it, the subnets name in foreman and ipam should be sync'd at the same time.

Reply:
@julioconstanzo Do you have an opinion about changing the vlan names in the switches use `_` instead of `-`?

Reply:
regarding de - to _, just standardization; I'll update names so they match IPAM.

Reply:
It looks like the transliteration to underscores has been backed out, so I'll mark this conversation as resolved.